### PR TITLE
VACMS-9649: Reset location values on toggle of event location type.

### PR DESCRIPTION
**Pull request by dsasser**
## Description

Resolves https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9649

## QA steps
Go to /node/add/event
Set the location type to "At a VA facility"
Fill in the facility location
Fill in the building number

Screen Shot 2022-09-01 at 9 21 28 AM

Without saving, switch the location type to "At a non-VA location"
Screen Shot 2022-09-01 at 9 21 49 AM
- [x] Verify the building, floor, or room field is left unchanged
- [x] Verify the address fields are all empty

Switch back to "At a VA facility" location type.
Screen Shot 2022-09-01 at 9 21 59 AM
- [x] Verify the building, floor, or room field is left unchanged
- [x] Verify the facility location value is empty

Switch the location type to "Online"
Fill in the URL
Screen Shot 2022-09-01 at 9 22 59 AM

Switch back to the "At a non-VA location" location type
Screen Shot 2022-09-01 at 9 23 17 AM
- [x] Verify all field values are empty

Switch back to the "At a VA facility" location type
Screen Shot 2022-09-01 at 9 23 24 AM
- [x] Verify all field values are empty

Switch back to the "Online" location type
Screen Shot 2022-09-01 at 9 23 31 AM
- [x] Verify url field is empty

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### CODE REVIEW

File: docroot/modules/custom/va_gov_events/js/eventFormHelpers.es6.js [hunk 1]
```diff
@@ -15,9 +15,66 @@
...
+  const fieldUrlOfOnlineEvent = document.getElementById(
+    "edit-field-url-of-an-online-event-0-uri"
+  );
+  const fieldUrlOfOnlineEventWrapper = document.getElementById(
+    "edit-field-url-of-an-online-event-wrapper"
+  );
+  const fieldAddressLine1 = document.getElementById(
+    "edit-field-address-0-address-address-line1"
+  );
+  const fieldAddressLin1Label = document.querySelector(
```
Comment (ndouglas) [inline docroot/modules/custom/va_gov_events/js/eventFormHelpers.es6.js:57]: `fieldAddressLin1Label` intentional?

Comment (dsasser) [reply docroot/modules/custom/va_gov_events/js/eventFormHelpers.es6.js:57]: Nay, good eye.

**Review by ndouglas — DISMISSED** [review]
Looks great!  The code changes make this file dramatically more readable and maintainable.  QA steps were clear and went through without a hitch.

Screen Shot 2022-09-01 at 12 39 52 PM

Do you have a couple hours to spare writing Cypress tests that replicate your QA steps?

**Comment by dsasser** [comment]
> Do you have a couple hours to spare writing Cypress tests that replicate your QA steps?
Sure I'll give it a shot! Are you open to moving the test writing to a separate ticket should it take me beyond a reasonable timeframe to write them?

**Comment by ndouglas** [comment]
Oh yeah, that's fine.  I'm just trying to expand automated test coverage, by begging/borrowing/stealing etc 😁

**Comment by dsasser** [comment]
Unfortunately I struggled with getting cypress working locally for too long and blew through the "reasonable" time allowance. Therefore I have created https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10605 to track adding the tests (which I completely agree should happen).

**Review by ndouglas — APPROVED** [review]
😭 ❤️

**Comment by ndouglas** [comment]
@dsasser Do you mind telling me a bit about your experience trying to get Cypress working locally?  I _gotta_ fix it if I want more tests 🙂

**Pull request merged**